astroid/nodes/scoped_nodes/scoped_nodes.py

```diff
@@ -1652,7 +1652,9 @@ def is_abstract(self, pass_is_abstract=True, any_raise_is_abstract=False):
         if pass_is_abstract:
             return True
 
-    def is_generator(self):
```

Comment:
`is_abstract` didn't return anything at the very end. Shouldn't Pylint have caught that?

Reply:
None is falsey, so it worked unless we called it as ``is_abstract(...) is False``. But yeah it should have raised an inconsistent return statement (astroid is not at the same level of quality than pylint, some message are disabled).